src/includes/snippets/RulesEngineExecution/en/snippet.mdx

```diff
@@ -0,0 +1,9 @@
+While behaviors and rules are executed in the order they are arranged, some types of behaviors cannot be stacked. 
+
+If a behavior of the type **Set**, such as **Set Custom Response** in Edge Firewall and **Set Origin** in Edge Application, is added multiple times to the rules in Rules Engine, only the last behavior from the last rule in which the criteria was met will be executed.
+
+Behaviors of the type **Add** are cumulative and can be added multiple times to rules. This means that if **Add Cookie** and **Add Header** are executed multiple times for the same key-value pairs, multiple identical entries will be added.
+
+However, for unique headers such as `Host`, the last behavior of type **Add Header** that is executed will overwrite the previous value, since there cannot be more than one `Host` header in a request. 
```

Comment:
```suggestion
However, for unique headers such as `Host`, the last behavior of type **Add Header** that is executed will overwrite the previous value, since there can't be more than one `Host` header in a request. 
```